<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormBaptis</class>
 <widget class="QWidget" name="FormBaptis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>381</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="idBaptisLabel">
      <property name="text">
       <string>Id Baptis</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="idBaptisLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="idJemaatLabel">
      <property name="text">
       <string>Id Jemaat</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="idJemaatLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="namaBaptisLabel">
      <property name="text">
       <string>Nama Baptis</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="namaBaptisLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="jenisKelaminLabel">
      <property name="text">
       <string>Jenis Kelamin</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="jenisKelaminComboBox">
      <item>
       <property name="text">
        <string>L</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>P</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="namaAyahLabel">
      <property name="text">
       <string>Nama Ayah</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="namaAyahLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="namaIbuLabel">
      <property name="text">
       <string>Nama Ibu</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="namaIbuLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cari</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableBaptis">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>381</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
